src/main/java/com/three_tech_solutions/slot_app/services/implementations/UserServiceImpl.java

```diff
@@ -20,4 +24,12 @@ public User loadUserByUsername(String username) throws UsernameNotFoundException
         return userRepository.findByUsername(username)
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Credenciales ingresadas incorrectas"));
     }
+    @Override
+    public List<Student> getUserStudents(UUID userId) {
+        return userRepository.findById(userId)
+                .map(user -> user.getStudents())//si existe el usuario,obtene los estudiantes
+                .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "El usuario no existe")); //si no hay ysuario, se lanza el error
```

Comment:
Si bien los comentarios a vos te sirven para que te quede claro, en realidad no deberiamos tenerlo en el codigo ya que no agregan informacion relevante (para alguien que sabe Java), asi que deberias quitarlos

```suggestion
                .map(user -> user.getStudents())
                .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "El usuario no existe"));
```